clients/node/client-rds-data-node/protocol/AwsRestJson1_1.ts

```diff
@@ -0,0 +1,297 @@
+import {
+  ColumnMetadata,
+  ExecuteStatementRequest,
+  ExecuteStatementResponse,
+  Field,
+  SqlParameter
+} from "../models/rdsdataservice";
+import { HttpRequest, HttpResponse } from "@aws-sdk/protocol-http";
+import * as __aws_sdk_stream_collector_node from "@aws-sdk/stream-collector-node";
+import * as __aws_sdk_util_utf8_node from "@aws-sdk/util-utf8-node";
+
+export function executeStatementAwsRestJson1_1Serialize(
+  input: ExecuteStatementRequest
+): HttpRequest {
+  let body: any = {};
```

Comment:
A performance improvement here would be to use a streaming JSON serializer rather than needing to store the JSON object in an intermediate value. This should be done at some point but isn't a blocker right now.